src/components/button/Button.js

```diff
@@ -381,5 +381,16 @@ export default class ButtonComponent extends BaseComponent {
   focus() {
     this.buttonElement.focus();
   }
-}
 
+  getView() {
+    if (this.component.displayInEditGrid) {
+      this.build();
```

Comment:
Why are you having to "build" the component here. ```getView``` will usually get called after the component has already been built.